.gitignore

```diff
@@ -2,4 +2,6 @@
 **/.idea/*
 .idea/*
 **/__pycache__/**
-.DS_Store
\ No newline at end of file
+.DS_Store
+icone_to_wzdx_test.geojson
```

Comment:
Since this is already part of the solution, you'll have to delete this file and commit that before the gitignore takes affect.

Reply:
just deleted the file

Reply:
Looks good!